blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties

```diff
@@ -0,0 +1,305 @@
+#请输入一个有效的URL
+creation.git.step1.repo_error_invalid=\u8bf7\u8f93\u5165\u4e00\u4e2a\u6709\u6548\u7684\u0055\u0052\u004c
+#请输入一个URL.
+creation.git.step1.repo_error_required=\u8bf7\u8f93\u5165\u4e00\u4e2a\u0055\u0052\u004c\u002e
+#仓库URL
+creation.git.step1.repo_title=\u4ed3\u5e93\u0055\u0052\u004c
+#连接到Git仓库
+creation.git.step1.title=\u8fde\u63a5\u5230\u0047\u0069\u0074\u4ed3\u5e93
+#保存
+creation.git.step2.button_save=\u4fdd\u5b58
+#'{0}'可用
+creation.git.step2.name_available={0}\u53ef\u7528
+creation.git.step2.name_required='{0}'\u53ef\u7528. \u9700\u8981\u4e00\u4e2a\u552f\u4e00\u7684\u540d\u5b57.
+creation.git.step2.name_unavailable='{0}'\u4e0d\u53ef\u7528. \u8bf7\u4f7f\u7528\u5176\u4ed6\u540d\u5b57.
+#Pipeline名字
+creation.git.step2.text_name_placeholder=\u0050\u0069\u0070\u0065\u006c\u0069\u006e\u0065\u540d\u5b57
+#命名冲突
+creation.git.step2.title=\u547d\u540d\u51b2\u7a81
+#打开
+creation.git.step3.button_open=\u6253\u5f00
+#完成
+creation.git.step3.title_completed=\u5b8c\u6210
+#正在创建Pipeline...
+creation.git.step3.title_pipeline_create=\u6b63\u5728\u521b\u5efa\u0050\u0069\u0070\u0065\u006c\u0069\u006e\u0065\u002e\u002e\u002e
+## github enterprise add server dialog
+#取消
+creation.githubent.add_server.button_cancel=\u53d6\u6d88
+#添加服务
+creation.githubent.add_server.button_create=\u6dfb\u52a0\u670d\u52a1
+#Jenkins需要知道你的GitHub Enterprise server地址.
+creation.githubent.add_server.instructions=\u004a\u0065\u006e\u006b\u0069\u006e\u0073\u9700\u8981\u77e5\u9053\u4f60\u7684\u0047\u0069\u0074\u0048\u0075\u0062\u0020\u0045\u006e\u0074\u0065\u0072\u0070\u0072\u0069\u0073\u0065\u0020\u0073\u0065\u0072\u0076\u0065\u0072\u5730\u5740\u002e
+#服务地址已存在.
+creation.githubent.add_server.text_name_error_duplicate=\u670d\u52a1\u5730\u5740\u5df2\u5b58\u5728\u002e
+#请输入名字.
+creation.githubent.add_server.text_name_error_required=\u8bf7\u8f93\u5165\u540d\u5b57\u002e
+#我的GitHub Server
+creation.githubent.add_server.text_name_placeholder=\u6211\u7684\u0047\u0069\u0074\u0048\u0075\u0062\u0020\u0053\u0065\u0072\u0076\u0065\u0072
+#服务名
+creation.githubent.add_server.text_name_title=\u670d\u52a1\u540d
+#URL已经被其他服务使用
+creation.githubent.add_server.text_url_error_duplicate=\u0055\u0052\u004c\u5df2\u7ecf\u88ab\u5176\u4ed6\u670d\u52a1\u4f7f\u7528
+#无法连接到可用的GitHub server.
+creation.githubent.add_server.text_url_error_invalid=\u65e0\u6cd5\u8fde\u63a5\u5230\u53ef\u7528\u7684\u0047\u0069\u0074\u0048\u0075\u0062\u0020\u0073\u0065\u0072\u0076\u0065\u0072\u002e
+#请输入有效的URL.
+creation.githubent.add_server.text_url_error_required=\u8bf7\u8f93\u5165\u6709\u6548\u7684\u0055\u0052\u004c\u002e
+creation.githubent.add_server.text_url_placeholder=https://github.example.com
+creation.githubent.add_server.text_url_title=Server URL
+#添加GitHub Server
+creation.githubent.add_server.title=\u6dfb\u52a0\u0047\u0069\u0074\u0048\u0075\u0062\u0020\u0053\u0065\u0072\u0076\u0065\u0072
+## github enterprise creation flow
```

Comment:
If the comments can be translated, then this string should be translated as well.